controllers/pane-new-page.js

```diff
@@ -0,0 +1,30 @@
+var edit = require('../services/edit'),
+  dom = require('../services/dom'),
+  progress = require('../services/progress');
+
+module.exports = function () {
+  function constructor(el) {
+    this.el = el;
+    this.form = dom.find(el, '.select-page-type');
+  }
+
+  constructor.prototype = {
+    events: {
+      '.create-article-page click': 'onNewArticle'
+    },
+
+    onNewArticle: function (e) {
+      e.preventDefault();
+
+      return edit.createPage()
+        .then(function (url) {
+          location.href = url;
+        })
```

Comment:
L19-L22 is the same as https://github.com/nymag/clay-kiln/blob/new-page-type-selector/controllers/kiln-toolbar.js#L32-L34